labelbox/client.py

```diff
@@ -931,6 +931,90 @@ def delete_unused_feature_schema(self, feature_schema_id):
                 "Failed to delete the feature schema, message: " +
                 response.json()['message'])
 
+    def update_feature_schema_definition(self, **kwargs):
+        feature_schema_id = kwargs.get("feature_schema_id")
+        title = kwargs.get("title")
+        endpoint = self.rest_endpoint + "/feature-schemas/" + feature_schema_id + '/definition'
+        response = requests.patch(
+            endpoint,
+            headers=self.rest_endpoint_headers,
+            json={"title": title},
+        )
+
+        if response.status_code == 200:
+            return response.json()
+        else:
+            raw_message = response.json()['message']
+            message = raw_message[0] if isinstance(raw_message,
+                                                   List) else raw_message
+            raise labelbox.exceptions.LabelboxError(
+                "Failed to update the feature schema's definition, message: " +
+                message)
+
+    def upsert_feature_schema(self, feature_schema):
```

Comment:
It's pretty unclear what should be passed into this function, and based on the code, user can pass in anything to the REST request, since we just dump it all as JSON. Could you add more validation, and maybe more specificity to the parameter here? It would be good to at least add a type definition to `feature_schema`, and validate all expected fields